apps/web/src/hooks/use-kyc-websocket.ts

```
@@ -0,0 +1,151 @@
+      ...prev,
+      reconnectAttempts: prev.reconnectAttempts + 1,
+      lastReconnectTime: Date.now(),
+    }));
+
+    currentReconnectDelay.current = Math.min(
+      currentReconnectDelay.current * 1.5,
+      MAX_RECONNECT_DELAY
+    );
+  };
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: handleReconnect

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)